provider_shopper/lib/screens/cart.dart

```diff
@@ -85,11 +85,18 @@ class _CartTotal extends StatelessWidget {
             SizedBox(width: 24),
             TextButton(
               onPressed: () {
-                Scaffold.of(context).showSnackBar(
-                    SnackBar(content: Text('Buying not supported yet.')));
+                //Updated Method of using Snack Bar for more info visit "https://flutter.dev/docs/release/breaking-changes/scaffold-messenger"
```

Comment:
You can leave this comment out. If we added links every time the API changed, we'd have run out of room for the actual code by now. 😄